### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Sibusiso-Retro**

### CODE REVIEW

File: SkietbaanBE/SkietbaanBE/Controllers/UserController.cs [hunk 1]
```diff
@@ -10,50 +14,70 @@ namespace SkietbaanBE.Controllers
...
-            _context.Add(user);
-            _context.SaveChanges();
+        public async Task<HttpResponseMessage> Post([FromBody] User user)
+        {
+            
+            if (ModelState.IsValid)
+            {
+                //get today's date and save it under user entry date
+                DateTime currentDate = new DateTime();
+                user.EntryDate = currentDate;
```
Comment (BWMuller) [inline SkietbaanBE/SkietbaanBE/Controllers/UserController.cs]: Why is currentDate a new variable? Just assign it directly

File: SkietbaanBE/SkietbaanBE/Controllers/UserController.cs [hunk 2]
```diff
@@ -10,50 +14,70 @@ namespace SkietbaanBE.Controllers
...
+            {
+                return new OkObjectResult("user does not exist");
+            }
 
-        // DELETE: api/ApiWithActions/5
-        [HttpDelete]
-        public void Delete(User user) {
-            _context.Users.Remove(user);
-            _context.SaveChanges();
+            _context.Users.Update(user);
```
Comment (BWMuller) [inline SkietbaanBE/SkietbaanBE/Controllers/UserController.cs]: Where is your check if the user actually exists in the db? You have a check to ensure the user has an id but not if he actually exists

Comment (Sibusiso-Retro) [reply SkietbaanBE/SkietbaanBE/Controllers/UserController.cs]: missed it, resolving now

File: SkietbaanBE/SkietbaanBE/Controllers/UserController.cs (hunk 1 above)
Comment (Sibusiso-Retro) [reply SkietbaanBE/SkietbaanBE/Controllers/UserController.cs]: getting today's date

Comment (BWMuller) [reply SkietbaanBE/SkietbaanBE/Controllers/UserController.cs]: `user.EntryDate = new DateTime(); `
Same effect, no surplus of variables

**Pull request closed without merging**